server/main.go

```diff
@@ -103,9 +104,12 @@ func main() {
 		glog.Fatalln("Cannot config global logger, use default one: ", zap.Error(err))
 	}
 
+	if cfg.Log.StderrRedirectFile != nil {
+		config.Default.System.SystemLogDBPath = filepath.Dir(*cfg.Log.StderrRedirectFile)
+	}
 	defer func() {
 		if r := recover(); r != nil {
-			recovery.CrashLog(r, cfg.Log)
+			recovery.CrashLog(r)
 		}
```

Comment:
create a function in recovery package:
```
func Recover() {
    if r := recover(); r != nil {
        CrashLog(r)
    }
}
```
such that you can call `defer recovery.Recover()`

Reply:
ok